helm/cluster-cloud-director/files/etc/gs-kube-proxy-patch.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+#
+# (PK) I couldn't find a better/simpler way to conifgure it. See:
+# https://github.com/kubernetes-sigs/cluster-api/issues/4512
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
```

Comment:
It is run once and this option gives us to understand what happened by checking cloud-init logs.